libraries/standard/json/src/json.c

```diff
@@ -0,0 +1,1014 @@
+#include <assert.h>
+#include <ctype.h>
+#include <stddef.h>
+#include <stdint.h>
+#include "json.h"
+
+typedef enum bool
+{
+    true = 1, false = 0
+} bool;
```

Comment:
I think it would be good to choose a name that is not a keyword in C++. Being C++ compatible isn't a requirement, but in this case it seems easy enough to do, and it might save someone a porting headache down the line

Reply:
changed